src/Components/Blueprints/BuildImagesButton.tsx

```diff
@@ -1,36 +1,139 @@
-import React from 'react';
+import React, {useState} from 'react';
 
-import { Button, ButtonProps } from '@patternfly/react-core';
+import {
+  ButtonProps,
+  Dropdown,
+  MenuToggle,
+  MenuToggleElement,
+  Menu,
+  MenuContent,
+  MenuList,
+  MenuItem, Flex, FlexItem, Spinner, MenuToggleAction,
+} from '@patternfly/react-core';
 
 import { selectSelectedBlueprintId } from '../../store/BlueprintSlice';
 import { useAppSelector } from '../../store/hooks';
-import { useComposeBlueprintMutation } from '../../store/imageBuilderApi';
+import {
+  ImageTypes,
+  useComposeBlueprintMutation,
+  useGetBlueprintQuery,
+} from '../../store/imageBuilderApi';
+import {skipToken} from "@reduxjs/toolkit/query";
 
-type BuildImagesButtonPropTypes = {
-  variant?: ButtonProps['variant'];
-  children?: React.ReactNode;
-};
-
-export const BuildImagesButton = ({
-  variant,
-  children,
-}: BuildImagesButtonPropTypes) => {
+export const BuildImagesButton = () => {
   const selectedBlueprintId = useAppSelector(selectSelectedBlueprintId);
+  const [selectedTargets, setSelectedTargets] =useState<ImageTypes[]>([]);
```

Comment:
IMO this means it's all unselected by default, IMO easier would be to
```suggestion
  const [selectedTargets, setSelectedTargets] =useState<ImageTypes[] | undefined>(undefined);
```